src/guide/bash.md

```diff
@@ -374,10 +374,11 @@ Account {
     },
     assets: {},
     signatories: [
-    PublicKey {
-        digest_function: "ed25519",
-        payload: "A753146E75B910AE5E2994DC8ADEA9E7D87E5D53024CFA310CE992F17106F92C",
-    },
+        PublicKey {
```

Comment:
I should to say then in the last version this output looks different. For example:
```
Account {
        id: Id {
            name: "genesis",
            domain_id: Id {
                name: "genesis",
            },
        },
        assets: {},
        signatories: {
            { digest: ed25519, payload: 4CFFD0EE429B1BDD36B3910EC570852B8BB63F18750341772FB46BC856C5CAAF },
        },
        permission_tokens: {},
        signature_check_condition: SignatureCheckCondition(
            EvaluatesTo {
                expression: ContainsAny(
                    ContainsAny {
                        collection: EvaluatesTo {
                            expression: ContextValue(
                                ContextValue {
                                    value_name: "transaction_signatories",
                                },
                            ),
                            _value_type: PhantomData,
                        },
                        elements: EvaluatesTo {
                            expression: ContextValue(
                                ContextValue {
                                    value_name: "account_signatories",
                                },
                            ),
                            _value_type: PhantomData,
                        },
                    },
                ),
                _value_type: PhantomData,
            },
        ),
        metadata: Metadata {
            map: {},
        },
        roles: {},
    },
```